app/models/user.rb

```diff
@@ -19,7 +19,17 @@ class User < ApplicationRecord
   scope :halfyear_actives, ->(time = Time.now) { logged_in_since(time - 6.month) }
   scope :active, -> { joins(:person).where(people: {closed_account: false}) }
 
-  devise :database_authenticatable, :registerable,
+  attribute :otp_secret
+
+  devise :two_factor_authenticatable,
+         otp_secret_encryption_key: "c0f2d6a8fa2873e40be5768f99b99260a3b0462ad9dd3f3\
+71432751b4e632f6ab4519b97dc7fdf5fa90c4dfdffa3ed0a917cd7a51382bac8b26166a5540bc603"
+
+  devise :two_factor_backupable,
+         otp_backup_code_length: 16,
```

Comment:
Layout/AlignHash: Align the elements of a hash literal if they span more than one line.